modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APISchemaRelevantObjectEntryModelListenerTest.java

```diff
@@ -57,6 +63,18 @@ public void test() throws Exception {
 			).toString(),
 			"headless-builder/applications", Http.Method.POST);
 
+		JSONObject apiApplicationPublishedJSONObject =
+			HTTPTestUtil.invokeToJSONObject(
+				JSONUtil.put(
+					"applicationStatus", "unpublished"
```

Comment:
it should be `published` due to the name is `apiApplicationPublishedJSONObject`.